### DISCUSSION TITLE
Parte 1: test hard y extreme

### DISCUSSION THREAD

**Discussion by domiSoto** [IIC2133-PUC/2024-2#129] [Tarea 1] [answered]
Hola, que podría estar mal con mi implementación de la parte 1 si paso sin errores los test easy y medium pero tengo bastantes errores en los demás? Intente debuggear pero no he encontrado el error, que casos borde se prueban principalmente en esos test? Cualquier ayuda me serviría 😭

**Comment by tomastrivino** [comment, marked as answer]
No podemos decirte qué es lo que está mal con tu código. Te recomendaría usar valgrind para analizar los posibles leaks y darle una vuelta a tu código, seguramente hay algún caso borde en que se cae o no cumple la complejidad.
